src/docs/product/sentry-basics/search/saved-searches.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: Saved Searches
+sidebar_order: 20
+keywords: ["saved search", "premade searches"]
+description: "Learn more about recommended, pinned, and saved searches."
+---
+
+In the "Saved Searches" tab of **Issues**, you can access recommended searches, pin searches that you find most useful, and take advantage of organization-wide saved searches.
+
+## Recommended Searches
+
+Recommended searches are common search terms that we think you're likely to use. These premade searches are listed under "Recommended Searches" in the "Saved Searches" dropdown, and they're listed in order of when you've most recently used them.
+
+![Open Saved Searches dropdown on the Issues page.](search-premade-dropdown.png)
+
+## Pinned Searches
+
+You can pin a search, and it will become the default view you see on the **Issues** page. The pinned search is only visible to you and is relevant across your projects.
+
+1. Type a search into the search bar.
+
+   ![The search bar of the Issues page with two search terms.](search-pinning-step-one.png)
+
+2. Click the pin icon next to that search.
+
+   ![Cursor mousing over the pin icon and tool-tip displayed.](search-pinning-step-two.png)
+
+3. Once pinned, Sentry will add the search to the "Saved Searches" dropdown. The search label in the text will read, "My Pinned Search".
+
+   ![Pinned search in Saved Searches dropdown.](search-pinning-step-three.png)
+
+### Changing a Pinned Search
+
+To change your pinned search, do the following:
+
+1. Select your pinned search. Un-click the pin icon. Your default search will return to `is:unresolved`.
+
+2. Run another search. Click the pin icon. The query listed as "My Pinned Search" will now be the new pinned query, instead of the original one.
+
+### Pinning a Recommended Search
+
+You can pin a recommended search the same way you pin any other search. When you've selected a recommended search, and the recommended search query populates the search bar, pin it.
+
+## Organization Wide Saved Searches
+
```

Comment:
does the lack of text here cause issues with the RH display?

Reply:
No, it doesn't cause issues--that's the way it is in the current docs and it renders fine. In my experience, it's just skipped heading levels that cause rendering issues.